packages/teraslice/src/lib/workers/execution-controller/index.ts

```diff
@@ -916,6 +917,11 @@ export class ExecutionController {
                         this.exId
                     } to finish...`
                 );
+                this.logger.debug(`Vars at timeout:\nisExecutionDone: ${this.isExecutionDone}\nclient.ready: ${this.client.ready}\n`
```

Comment:
Maybe call this Execution Controller state vars? Depending on how the user is viewing the logs, they may not know the context. It will make it more explicit 
```suggestion
                this.logger.debug(`Execution controller state vars at timeout:\nisExecutionDone: ${this.isExecutionDone}\nclient.ready: ${this.client.ready}\n`
```